### PR TITLE
add skeleton for the item count

### LINKED ISSUE #5893: Loading skeleton for the item count
https://github.com/kodadot/nft-gallery/assets/90852205/cb666ec5-f42a-48fa-93ae-e58a33af42ee

so right now, after #5872 the preview always jumps when the actual content is loaded. 
Thus it would be nice to have loading skeleton for the item count as well. 

good follow up for @leo-anderson-x

### PR DESCRIPTION
**Pull request by leo-anderson-x**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Feature

## Context

- [x] Closes #5893 

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://github.com/kodadot/nft-gallery/assets/128157824/16c506b4-5072-4b8d-9685-fbc17e98a819

### <samp>🤖 Generated by Copilot at e7cd8e4</samp>

This pull request enhances the loading states of the `CollectionList` and `ItemsGrid` components by using a `NeoSkeleton` component from a library and customizing its style. This improves the user experience and the visual consistency of the app.

### <samp>🤖 Generated by Copilot at e7cd8e4</samp>

> _`ItemsGrid` shows_
> _`NeoSkeleton` while loading_
> _Winter bones in style_

### CODE REVIEW

File: components/collection/CollectionList.vue [hunk 1]
```diff
@@ -168,3 +172,9 @@ watch(
   }
 )
 </script>
+
+<style lang="scss" scoped>
+.skeleton-fixed-width {
+  width: 80px;
```
Comment (roiLeo) [inline components/collection/CollectionList.vue:178]: There is already a `width` prop on `NeoSkeleton`, have you tried something like: 
```ts
<NeoSkeleton v-else-if="isLoading" no-margin width="80px" />
```

File: components/items/ItemsGrid/ItemsGrid.vue [hunk 2]
```diff
@@ -88,3 +93,9 @@ onBeforeMount(async () => {
   isLoading.value = false
 })
 </script>
+
+<style lang="scss" scoped>
+.skeleton-fixed-width {
+  width: 80px;
+}
```
Comment (roiLeo) [inline components/items/ItemsGrid/ItemsGrid.vue:100]: same comment

**Review by roiLeo — COMMENTED** [review]
small stuff, otherwise code lgtm

File: components/collection/CollectionList.vue (hunk 1 above)
Comment (leo-anderson-x) [reply components/collection/CollectionList.vue:178]: @roiLeo I tried, it doesn't work. `width` applied to the skeleton item width, but width of skeleton container is still 100% that will cause some odd style issue.
CleanShot 2023-05-11 at 18 20 51@2x

It is true that there are other ways to solve this problem: for example, making the filter tag on the left side not wrap, and then making the skeleton item on the right side right-aligned. But the easiest way is to give NeoSkeleton a separate container and specify its width.

Comment (roiLeo) [reply components/collection/CollectionList.vue:178]: Understandable, maybe rename class to skeleton-container-fixed-width

**Review by roiLeo — APPROVED** [review]

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 24.74 USD/KSM ~ 0.404 $KSM
🧗 [DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C](https://kusama.subscan.io/account/DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C)
🔗 [0xa21f5ff28c8da3bb81c56703b5855718208d46019e9950cedca17219c74ad17f](https://kusama.subscan.io/extrinsic/0xa21f5ff28c8da3bb81c56703b5855718208d46019e9950cedca17219c74ad17f)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**